sdk/python/foundation-models/system/finetune/image-classification/multiclass-classification/hftransformers-fridgeobjects-multiclass-classification.ipynb

```diff
@@ -505,13 +505,14 @@
     "    # model_selection_args\n",
     "    \"model_family\": \"HuggingFaceImage\",\n",
     "    # # specify the model_name instead of mlflow_model if you want to use a model from the huggingface hub\n",
-    "    \"mlflow_model\": foundation_model,\n",
-    "    # \"model_name\": huggingface_model_name,\n",
+    "    # \"mlflow_model\": foundation_model,\n",
```

Comment:
Can we use the foundation model instead of HuggingFace model in the example? We have the models published now.